scenarioo-server/src/main/java/org/scenarioo/rest/builds/BuildsImporterResource.java

```diff
@@ -98,6 +95,35 @@ public void reimportBuild(@PathParam("branchName") final String branchName,
 		ScenarioDocuBuildsManager.INSTANCE.reimportBuild(buildIdentifier);
 	}
 
+	@GET
+	@Path("importBuild/{branchName}/{buildName}/{comparisonBranchName}/{comparisonBuildName}/{comparisonName}")
+	@Produces({"application/json"})
+	public Response importBuild(
```

Comment:
@til77 the URL to get the diff info for a comparison, is: http://demo.scenarioo.org/scenarioo-diff-viewer-plus/rest/diffViewer/baseBranchName/{put-your-branch-name-here}/baseBuildName/{put-your-build-name-here}/comparisonName/{put-the-name-of-the-comparison-here}/buildDiffInfo  -returned json contains the result in field "changeRate" as a decimal number